tests/libvmi/operations.go

```diff
@@ -0,0 +1,31 @@
+package libvmi
+
+import (
+	. "github.com/onsi/gomega"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
+)
+
+func CreateVmi(client kubecli.KubevirtClient, namespace string, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
+	vmi, err := client.VirtualMachineInstance(namespace).Create(vmi)
+	if err != nil {
+		return nil, err
+	}
+	return vmi, nil
+}
+
+func DeleteVmi(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) error {
```

Comment:
Please order the functions such that the caller is before the callee. It helps reading the code from the most abstract down the details, without the need to bounce all the time.

Also, if no one is using this, keep it unexported. or, perhaps just do not implement `DeleteVmiAndWaitForDeletion`, and let the caller do it in two steps. Up to you.